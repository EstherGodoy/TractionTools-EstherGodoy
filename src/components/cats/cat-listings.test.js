import React from 'react';
import ReactDOM from 'react-dom';
import CatListings from './cat-listings';
import renderer from 'react-test-renderer';

//renders
it('component renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<CatListings />, div);
});

//snapshot test
test("cat listings snapshot test", (() => {
  const component = renderer.create(
    <CatListings/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
}))
