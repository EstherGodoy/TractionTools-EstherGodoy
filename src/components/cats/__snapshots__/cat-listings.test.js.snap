// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cat listings snapshot test 1`] = `
<section>
  <ul
    className="cats"
  />
  <nav
    className="pagination"
  >
    <button
      className="prev"
      disabled={true}
      onClick={[Function]}
    >
      Previous Page
    </button>
    <button
      className="next"
      disabled={false}
      onClick={[Function]}
    >
      Next Page
    </button>
  </nav>
</section>
`;
